### ISSUE TITLE
Testing getting started / quick start documentation

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/boiler#609]
#### Description / User Story

After #364 and #365 are done, we have to test if the setup / getting started documentation works.

----
#### Requirements
- [x] #364 
- [x] #365 

----
#### Acceptance Criteria
- [x] The getting started / quick start documentation has been tested
- [x] The documentation has been improved based on the findings
- [x] When someone goes through all the steps in the setup / getting started documentation they will have a working duplicate of the repository and the figma file, which both are connected via a token system

----
#### Background information
- Changing the font or the icon set of the duplicate is something we will provide at a later point, but not in the alpha release
